app/components/app.jsx

```diff
@@ -1,3 +1,4 @@
+import { connect }                       from 'react-redux';
```

Comment:
when i toggle night mode it posts this error in the console:

`Warning: [react-router] You cannot change <Router routes>; it will be ignored`

i think due to this file? but i'm not sure.

Reply:
I think this is because when you change nightmode, it'll rewrite the whole app including the router, but router doesn't support changing the routes after the initial time. 

It might be better to have the `night-mode` class applied to the individual components rather than the overall one. You could create a small wrapper component to wrap each of the `HomeComponent`, `LoginComponent` etc that would do the nightmode so you aren't duplicating the code.